docs/index.rst

```diff
@@ -270,6 +270,73 @@ And ``templates/mytemplate.pt`` might look like so:
 .. index::
    single: Chameleon text templates
 
+
+Using a master page
+~~~~~~~~~~~~~~~~~~~
+
+You can also use macros and slots to create a master page that can be used by all your templates. This is very similar to using a macro from another another template but uses the `IBeforeRender` event subscriber to make the macros availlable to any template.
```

Comment:
* Wrap to 79 character column width.
* Remove extra "another"
* Correct spelling of "availlable", has an extra "l".